src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -43,15 +48,40 @@ public static String getAccessToken(User user) {
     return getAccessToken(user.getUserName(), user.getPassword());
   }
 
+  public static void deleteAllTokens(User user) {
+    String heliosGetTokenURL = HeliosConfig.getUrl(HeliosConfig.HeliosController.USERS);
+
+    CloseableHttpClient httpClient =
+        HttpClientBuilder.create().disableCookieManagement().disableConnectionState()
+            .disableAutomaticRetries().build();
+
+    HttpDelete httpDelete =
+        new HttpDelete(String.format("%s/%s/tokens", heliosGetTokenURL, user.getUserId()));
+    httpDelete.setConfig(requestConfig);
+    httpDelete.setHeader("THE-SCHWARTZ", Configuration.getCredentials().apiToken);
+
+    CloseableHttpResponse response = null;
+    try {
+      response = httpClient.execute(httpDelete);
+
+      PageObjectLogging.log("DELETE HEADERS: ", response.toString(), true);
+
+    } catch (ClientProtocolException e) {
+      PageObjectLogging.log("CLIENT PROTOCOL EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+      throw new WebDriverException(e);
+    } catch (IOException e) {
+      PageObjectLogging.log("IO EXCEPTION",
+          IOEXCEPTION_ERROR_MESSAGE + ExceptionUtils.getStackTrace(e), false);
+      throw new WebDriverException(e);
+    }
+  }
+
   public static String getAccessToken(String userName, String password) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getAccessToken" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)